src/driver.py

```diff
@@ -113,6 +113,9 @@ def setup_browser(self):
                 break
             except Exception as e:
                 print (e)
+                time.sleep(0.5)
```

Comment:
Is this needed?

Reply:
I thought it is useful not to make many threads open browsers at once, but I think it's not useful for that. I removed this.